### ISSUE TITLE
Person Names

### ISSUE THREAD

**Issue by vanaukenk** [WormBase/website#6458]
Hello

On several occasions I’ve had problems in compiling non-redundant bibliographies because of a confusion of names, often Chinese names.

As a specific example, https://wormbase.org/resources/paper/WBPaper00048704

"Li C et al. (2015) PLoS Genet” OK, but "Chun, L., Hisamoto, N., & Matsumoto, K. (2015). Axon Regeneration Is….” not OK, seehttp://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005603

And on the person page https://wormbase.org/resources/person/WBPerson11816

Li Chun, "Also known as: Chun Li”

1. The name is “Chun Li”

2. "Also known as" does not seem appropriate.

I have not attempted to search systematically for this type of problem.

Thanks

Jonathan

**Comment by vanaukenk** [comment]
Hi Johnathan,

while I agree that the redundancy in names could be better, according to her lab site: http://bunshi3.bio.nagoya-u.ac.jp/~bunshi6/matsumoto_japanese/member.html
She put in 李 as her first name and 春 as her last which would be Li Chun, so it is in line with what wormbase currently shows.

Michael

**Comment by vanaukenk** [comment]
Hi Jonathan -

To add to Michael’s comment:

This is something I’ve confronted often doing  analyses of the literature and publishing trends over the years. I feel your pain. 

While I agree in this case “Also known as” isn’t exactly accurate, in its broader use it usually is. It is used to capture variations in namepresentation in publication: TW Harris, Todd Harris, etc. I suppose it could read “Also publishes as” but that won’t be 100% accurate either. Suggestions?

As we move to greater reliance on things like ORCid this situation should improve. 

Todd

**Comment by vanaukenk** [comment]
Hello Todd,

“Also erroneously known as” ;-)

It’s obviously not straightforward. Pubmed too gets it wrong sometimes, even when it is correct in an article [I had a student called Le He and hisname got mixed up everytime].

As I alluded to in a follow-up message to Michael, perhaps avoiding having "Li C et al.” and "Chun, L., Hisamoto, N….” on the same page (WBPaper00048704) would be a start.

Thanks

Jonathan

**Comment by vanaukenk** [comment]
This seems like something Cecilia should be able to accommodate in
Person and Author curation. I like an "Also erroneously known as" tag
or something comparable.

Todd, I'll let you bring her and Juancarlos into the conversation.
Karen
~~~~
Karen Yook

**Comment by vanaukenk** [comment]
Hi:

1.-  Paper class should be addressed by Kimberly.

2.- Person class.
     We capture names that authors published under and are known as.
	•	Chun Li ,
	•	Naoki Hisamoto ,
	•	Kunihiro Matsumoto
	•	https://doi.org/10.1371/journal.pgen.1005603
https://wormbase.org/resources/person/WBPerson11816#0132--10
Li Chun
aka Chun Li
	•	
 

http://bunshi3.bio.nagoya-u.ac.jp/~bunshi6/matsumoto_japanese/member.html
Li Chun

- We follow standard names: First name + Last name.
- I do not agree with the term “Also erroneously known as” .
- Names that are commonly use in each country does not mean it is erroneous.
- We use country names by: https://www.state.gov/misc/list/index.htm#h
It does not mean that Who int  or UN country lists are erroneous.

Have a great day!

cecilia

**Comment by vanaukenk** [comment]
It is incorrect to display the name as "Chun, L." here <https://wormbase.org/resources/paper/WBPaper00048704#04--10> since that is saying that her last name is "Chun".

We need to figure out how that came about.

Raymond

**Comment by ceciliaWBPerson1** [comment]
Hi, I emailed her to have her data clear. Waiting for reply.

cecilia

**Comment by vanaukenk** [comment]
@sibyl229 - this thread is about how we display author names in the Overview widget of the Paper pages.
@ceciliaWBPerson1 has contacted the user to confirm the correct first and last names, but depending upon the answer, we may need to re-visit how we display author names.

**Comment by vanaukenk** [comment]
@sibyl229 
@chris-grove has found another case where the display of a Person, rather than Author, name in the bibliographic citation on the paper page is actually giving an incorrect citation, i.e. the author's name changed since they published that paper.  See: https://wormbase.org/resources/paper/WBPaper00001923
To avoid any confusion, can we list the author name at the time of publication but still link out to the correct WBPerson page?

**Comment by sibyl229** [comment]
Hi @vanaukenk I think @azurebrd might have worked on the display code for Person. Let me ask him if he knows anything.

**Comment by azurebrd** [comment]
Ah, sure thing @sibyl229, I think this is about Paper object display, but I'll take a look.  So, @vanaukenk , we want to display the Author object value, but link to the Person object page ?  This is in the Overview widget, where it says "Link, E. M., & Thomas, J. H. (1994). A screen for nonconditional dauer-constitutive mutations in Caenorhabditis elegans. Genetics, 136, 879-86. " ?

**Comment by vanaukenk** [comment]
@azurebrd 
Yes, in the overview section of the paper page we could display the Author object value but link that value to the associated Person object page.  
So, for example, this paper:
https://www.wormbase.org/resources/paper/WBPaper00002485
Would display the first author as 'Malone EA' and link to the Person page for 'Elizabeth Link'.

**Comment by azurebrd** [comment]
@vanaukenk The first issue is resolved here https://juancarlos.wormbase.org/resources/paper/WBPaper00002485#04--10 if that looks good I'll make a pull request.  I'll look into the second issue (and may break things while poking at it, if it's not working at all, let me know and I'll stop while you test it)

**Comment by azurebrd** [comment]
@vanaukenk The second issue is also resolved here https://juancarlos.wormbase.org/resources/paper/WBPaper00044826#04--10  If that looks good, let me know to make a pull request.

**Comment by vanaukenk** [comment]
@azurebrd - the two examples above look fine now, but in testing some other papers I came across one where the author order in the overview does not match the author order in the paper or tree display:
https://juancarlos.wormbase.org/resources/paper/WBPaper00050843

**Comment by azurebrd** [comment]
@vanaukenk I'll take a look.  If you see anything else that looks wrong, let me know, thanks.

**Comment by azurebrd** [comment]
@vanaukenk Hi Kimberly.  The old way was getting the order based on the index of the author array, which works unless two authors share a name, which is why they were getting compressed into a single entry.  The way it's working now, there's a counter as it reads the author objects, but that doesn't work because they're not ordered. I'm thinking of going back to the index, but using a counter to track multiple entries for a given author, and then splicing them in somehow.  The thing is that acedb doesn't differentiate two authors of the same name, they get compressed into the same author object with separate person connections, so the order won't be the same as it was in the original paper.  Should all authors that share the same name be together, or can the second one (arbitrary second one ?) be tacked at the end ?   Presumably datomic could keep the authors as separate objects even if they have the same name, but going through acedb is going to flatten them.

**Comment by vanaukenk** [comment]
@azurebrd   Just returning to this one.

From what you've written, it doesn't seem like ACeDB leaves us with great options, as I doubt all authors that share the same name are sequential wrt author order, so if we want to display all authors in cases where there are duplicates, we're likely to get the order incorrect in some cases.

Can you confirm that these are our options:
1) Only display one of the authors when there is a duplicate author name
2) Display both authors in sequential order regardless of publication order
3) Display both authors with the first one in the correct order and the second tacked on at the end

I honestly don't know how widespread an issue this might be, but since author order generally has some meaning wrt contribution to the work, I'm really not sure what to do here.  

It almost seems as if we need to keep track of what papers correspond to what version of Possibly_publishes_as in WBPerson and then use that to display author names, but that sounds complicated.....
